### PR TITLE
Jetty 12.0.x cleanup contribution guide

### PR DESCRIPTION
**Pull request by gregpoulos**

### CODE REVIEW

**Comment by joakime** [comment]
@gregpoulos FYI, the Eclipse website for Jetty has changed.

It is now `https://eclipse.dev/jetty/` 

That means any of the old references that start with the following need to be updated ...

```
http://www.eclipse.org/jetty/
http://eclipse.org/jetty/
http://jetty.eclipse.org/
https://www.eclipse.org/jetty/
https://eclipse.org/jetty/
https://jetty.eclipse.org/
```

**Comment by gregpoulos** [comment]
> @gregpoulos FYI, the Eclipse website for Jetty has changed.
> 
> It is now `https://eclipse.dev/jetty/`
> 
> That means any of the old references that start with the following need to be updated ...
> 
> ```
> http://www.eclipse.org/jetty/
> http://eclipse.org/jetty/
> http://jetty.eclipse.org/
> https://www.eclipse.org/jetty/
> https://eclipse.org/jetty/
> https://jetty.eclipse.org/
> ```

Noted. I'll work on updating these references.

File: CONTRIBUTING.md
Comment (gregpoulos) [inline CONTRIBUTING.md:1]: I've moved a lot of this content into the AsciiDoc guide, to minimize duplicated content and consolidate information into a single place.

File: README.md
Comment (gregpoulos) [inline README.md:1]: I moved a substantial amount of development-related content out of here and into the Contribution Guide.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,62 +1,33 @@
 Contributing to Jetty
 =====================
```
Comment (sbordet) [inline CONTRIBUTING.md:2]: Not sure what these `=` signs mean... If they indicate a header, can we normalize with the rest of the docs? I.e.:

```markdown
# Contributing to Jetty
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,62 +1,33 @@
...
 
-- [https://projects.eclipse.org/projects/rt.jetty](https://projects.eclipse.org/projects/rt.jetty)
+Here are some quick links to other useful resources:
 
-Developer resources
---------------------
-Information regarding source code management, builds, coding standards, and more.
+* [**Source code.**](https://github.com/eclipse/jetty.project) Jetty's canonical git repository is located on GitHub at https://github.com/eclipse/jetty.project.
+
+* [**Mailing list.**](https://dev.eclipse.org/mailman/listinfo/jetty-dev) The [` jetty-dev@eclipse.org`](https://dev.eclipse.org/mailman/listinfo/jetty-dev) mailing list is a forum for technical discussion.
```
Comment (sbordet) [inline CONTRIBUTING.md:11]: Change the URLs to `https://accounts.eclipse.org/mailing-list/jetty-dev` and similar.

File: README.md [hunk 3]
```diff
@@ -1,22 +1,14 @@
...
-
-Submitting a patch or pull request?
-
-Make sure you have an Eclipse Contributor Agreement (ECA) on file.
-
-- [eclipse.org/legal/ecafaq](https://www.eclipse.org/legal/ecafaq.php)
-
-## Project description
+Eclipse Jetty
+=============
```
Comment (sbordet) [inline README.md:2]: Fix headers here too.

**Review by sbordet — APPROVED** [review]

**Comment by sbordet** [comment]
Merged into `jetty-12.0.x` in 02fc4392cc897e389e30683e24e7ff0515f279b7, ecf8795aef8730baa7dd965f0322fb01e6ba0e95 and e71fa6dedd862aeea05f2ef54dc5789dddef7971.

Thanks @gregpoulos!

**Comment by olamy** [comment]
Can we keep some links to how to build from the README file.
Usually people coming to github are here because they want to look at sources and probably even build the project from sources.
I would prefer we keep an easy way to find build instructions from the README.

**Comment by gregpoulos** [comment]
@olamy I'm happy to restore some of that info to the README – I'll open a separate PR.

**Pull request closed without merging**